### PR TITLE
Allow Stack to build using Common packages

### PR DESCRIPTION
**Pull request by MiYanni**
## Description

Changes to Stack:
- Moved modules out of nested `Stack` directories into `StackAdmin`
- Moved `Stack.sln` into `StackAdmin`
- Removed all common project references for Stack modules and StackAdmin modules
- Re-added Stack as part of All in our build.proj and associated tools scripts

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1080/

File: src/ResourceManager/Storage/Commands.Management.Storage.Test/Commands.Management.Storage.Test.csproj [hunk 1]
```diff
@@ -61,10 +61,10 @@
     <None Include="ScenarioTests\Common.ps1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
-    <None Include="..\..\Common\Commands.ScenarioTests.ResourceManager.Common\AzureRM.Resources.ps1">
+    <!-- <None Include="..\..\Common\Commands.ScenarioTests.ResourceManager.Common\AzureRM.Resources.ps1">
```
Comment (MiYanni) [inline src/ResourceManager/Storage/Commands.Management.Storage.Test/Commands.Management.Storage.Test.csproj]: Delete this.

File: src/ServiceManagement/Services/Commands.Utilities/packages.config [hunk 2]
```diff
@@ -53,7 +53,7 @@
   <package id="System.Net.Requests" version="4.0.11" targetFramework="net45" />
   <package id="System.Spatial" version="5.8.2" targetFramework="net45" />
   <package id="WindowsAzure.ServiceBus" version="2.8.2" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="9.0.0" targetFramework="net452" />
+  <package id="WindowsAzure.Storage" version="8.1.1" targetFramework="net452" />
```
Comment (MiYanni) [inline src/ServiceManagement/Services/Commands.Utilities/packages.config]: This was a bug I had from the Common PR.

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj [hunk 3]
```diff
@@ -48,53 +49,47 @@
...
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.KeyVault.Core">
-      <HintPath>..\..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <!--    <Reference Include="Microsoft.Azure.Management.Storage, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\packages\Microsoft.Azure.Management.Storage.6.4.0-preview\lib\net452\Microsoft.Azure.Management.Storage.dll</HintPath>
-    </Reference>
--->
+    <!-- <Reference Include="Microsoft.Azure.Management.Storage, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (MiYanni) [inline src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj]: Remove this.

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/packages.config [hunk 4]
```diff
@@ -4,6 +4,6 @@
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net40" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net40" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net40" />
-  <package id="Microsoft.WindowsAzure.Management.Storage" version="5.1.1" targetFramework="net40" />
+  <package id="Microsoft.WindowsAzure.Management.Storage" version="6.0.1" targetFramework="net40" />
```
Comment (markcowl) [inline src/StackAdmin/Storage.Management/Commands.Management.Storage/packages.config]: is this intentional?

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj [hunk 5]
```diff
@@ -48,53 +49,47 @@
...
+      <HintPath>..\..\..\packages\Microsoft.Data.OData.5.6.4\lib\net40\Microsoft.Data.OData.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Data.Services.Client">
-      <HintPath>..\..\..\..\packages\Microsoft.Data.Services.Client.5.6.4\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.6.4\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.WindowsAzure.Management.Storage">
-      <HintPath>..\..\..\..\packages\Microsoft.WindowsAzure.Management.Storage.5.1.1\lib\net40\Microsoft.WindowsAzure.Management.Storage.dll</HintPath>
```
Comment (markcowl) [inline src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj]: ??

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/packages.config (hunk 4 above)
Comment (MiYanni) [reply src/StackAdmin/Storage.Management/Commands.Management.Storage/packages.config]: Nope. Changed it back and fixed a different packages.config issue.

File: src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj (hunk 5 above)
Comment (MiYanni) [reply src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj]: Not intentional. Set it back to 5.1.1.

**Pull request merged**